modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -247,3 +247,101 @@ mirror:
+      - name: stable-4.12 #<1>
+        minVersion: 4.12.1
+        maxVersion: 4.12.1
+      - name: stable-4.13
+        minVersion: 4.13.1
+        maxVersion: 4.13.1
+----
+<1> Existing version that is present in local storage. Since the updated imageset configuration file does not remove this value, the images are not pruned.
+
+=== Pruning the existing version and updating to a specific version
+
+You want to prune the existing version and mirror with the specific version of `4.13.1`.
+A local storage backend contains images of a  specific version.
+
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: /home/user/metadata
+mirror:
+  platform:
+    channels:
+      - name: stable-4.13 #<1>
+        minVersion: 4.13.1
+        maxVersion: 4.13.1
+----
+<1> Replacing by `stable-4.13` prunes the existing version.
+
+=== Pruning previous operator version and updating to newer version
+
+A local storage backend includes only the Red Hat Advanced Cluster Security for Kubernetes Operator, version `4.0.1` and later in the `stable` channel. 
+The following `ImageSetConfiguration` file prunes the existing version and update to the newer version.
+
+.Example `ImageSetConfiguration` file
+[source,yaml,subs=attributes+]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: /home/user/metadata
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
+      packages:
+        - name: rhacs-operator
+          channels:
+          - name: stable #<1>
+----
+<1> Removing `minVersion` updates the Operator to the newer version in the `stable` channel.
+
+[id="oc-mirror-image-set-examples-operator-pruning-versions_{context}"]
+==== Pruning the existing Operator and mirroring a new Operator
+
+A local storage backend includes only the Red Hat Advanced Cluster Security for Kubernetes Operator, versions starting at 4.0.1 and later in the `stable` channel. The following `ImageSetConfiguration` file prunes the Red Hat Advanced Cluster Security for Kubernetes Operator and mirrors a new operator of latest version in the `stable` channel.
+
+.Example `ImageSetConfiguration` file for update
+[source,yaml,subs=attributes+]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: /home/user/metadata
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
+      packages:
+        - name: <new-operator-name> #<1>
+          channels:
+          - name: stable
+----
+<1> Replacing `rhacs-operator` with `new-operator-name` prunes the Red Hat Advanced Cluster Security for Kubernetes Operator. To add new Operator without pruning the existing Operator, add a new Operator under `mirror.operators.packages` instead of replacing the existing Operator.
```

Comment:
It is true that replacing works, while removing the operator (if the goal is just to prune) doesn't!
I wish we could find a less confusing workaround for the scenario of pruning a specific operator, but for nowm I'm short on ideas.